packages/bsky/tests/views/subscriptions.test.ts

```diff
@@ -0,0 +1,68 @@
+import { AtUri, AtpAgent } from '@atproto/api'
+import { TestNetwork, SeedClient, basicSeed } from '@atproto/dev-env'
+import { ids } from '../../src/lexicon/lexicons'
+
+describe('subscriptions views', () => {
+  let network: TestNetwork
+  let agent: AtpAgent
+  let pdsAgent: AtpAgent
+  let sc: SeedClient
+
+  // account dids, for convenience
+  let alice: string
+  let bob: string
+
+  beforeAll(async () => {
+    network = await TestNetwork.create({
+      dbPostgresSchema: 'bsky_views_subscriptions',
```

Comment:
we wanna make sure we use a different `dbPostgresSchema` for this test suite than the one we use in `tests/subscriptions.test.ts`